src/formats/consts.ts

```diff
@@ -21,3 +21,5 @@ export const KNOWN_FORMATS: Array<[string, FormatLookup, string]> = [
   ['json-schema-draft7', isJSONSchemaDraft7, 'JSON Schema Draft 7'],
   ['json-schema-2019-09', isJSONSchemaDraft2019_09, 'JSON Schema Draft 2019-09'],
 ];
+
+export const KNOWN_RULESETS = ['spectral:oas', 'spectral:asyncapi'];
```

Comment:
@P0lip How would you feel making `generate-assets.ts` consuming these?

Reply:
and maybe `/src/cli/services/linter/utils/getRuleset.ts`?

Reply:
Makes sense to me. Will do